lib/services/elasticsearch.js

```diff
@@ -799,7 +807,23 @@ class ElasticSearch extends Service {
   getMapping(request) {
     const esRequest = initESRequest(request);
 
-    return this.esWrapper.getMapping(esRequest);
+    let includeKuzzleInfo = false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "includeKuzzleInfo" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1854)